courses/free-courses-en.md

```diff
@@ -491,6 +491,7 @@
 ### DevOps
 
 * [AWS DevOps Engineer Learning Plan](https://explore.skillbuilder.aws/learn/public/learning_plan/view/85/devops-engineering-learning-plan?la=sec&sec=lp) - AWS
+* [Continuous Delivery & DevOps](https://www.coursera.org/learn/uva-darden-continous-delivery-devops) - University of Virginia
```

Comment:
This looks like a scheduled course, which means we'd need to maintain it. We don't accept such listings.